web/nginx/snippets/letsencrypt.conf

```diff
@@ -0,0 +1,4 @@
+# ACME-challenge
```

Comment:
what's this file do?

Reply:
We are using [this method](https://certbot.eff.org/docs/using.html#webroot) to valid our let's encrypt cert by means of certbot. 

The way it works is, certbot creates a file, let's encrypt access this file, and our SSL is validated!

Since our [enhanced Nginx policy](https://github.com/CTrando/UCSD-Planner-Helper/blob/2c4bbeae802064527165c6eceae525cc3ce14183/web/nginx/snippets/general.conf#L11) will refuse all the access to "dot files", we need to create an additional rule to let cerbot pass.

And certbot will clean up everything after validation. So nothing will be in that directory.

Reply:
Ah I see, thanks for the explanation!